docker/start

```diff
@@ -51,6 +51,10 @@ wait_for_ipfs() {
 
 start_query_node() {
     export DISABLE_BLOCK_INGESTOR=true
+    # Set custom poll interval
+    if [ -n "$ethereum_polling_interval" ]; then
+        export ETHEREUM_POLLING_INTERVAL=$ethereum_polling_interval
+    fi
```

Comment:
It would be better to have this block in the 'main' part of the script so that we don't need it in three different places; it should go either before or after setting `PGAPPNAME`

Reply:
@lutter yes true, I've updated the script